### PR TITLE
Support C610 and audio Player by GStreamer

### PR DESCRIPTION
**Pull request by nianshao163**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by codingspirit — CHANGES_REQUESTED** [review]

File: .github/workflows/qualcomm-qcs610.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          gh secret set --env ECR ECR_PASSWORD --body ${{ steps.login-ecr.outputs.docker_password }} --repo ${{ github.repository }}
+
+  qualcomm:
+    environment: ECR
+    needs: ecr-prepare
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        container:
+          ["482862934379.dkr.ecr.us-east-1.amazonaws.com/qualcomm:latest"]
```
[the remark this answers is not in the document]
Comment (chenln1124) [reply .github/workflows/qualcomm-qcs610.yml:48]: Yes,  we have already uploaded it.

[the remark this answers is not in the document]
Comment (chenln1124) [reply .github/workflows/qualcomm-qcs610.yml:48]: > Have we uploaded this docker image to ECR?

Yes, we have already uploaded it.

File: 3rdparty/GSTREAMER/GSTREAMERGstCmd.h [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+#pragma once
+
+// video:
+// gst-launch-1.0 -e rtspsrc location=rtsp://192.168.0.60:554/stream0 latency=500 ! rtph264depay ! queue ! h264parse config-interval=1 ! mpegtsmux name=muxer !  tcpserversink host=127.0.0.1 port=8501 &
+// "tcpclientsrc host=127.0.0.1 port=8502 ! tsdemux name=muxer ! h264parse ! appsink name=sink"
```
[the remark this answers is not in the document]
Comment (chenln1124) [reply 3rdparty/GSTREAMER/GSTREAMERGstCmd.h:19]: > This way, no matter what source you use, as long as you send it to TCP 127.0.0.1, the webrtc program can receive it without changing the logic.
> Maybe rtspsrc, v4l2src ...

This way, no matter what source you use, as long as you send it to TCP 127.0.0.1, the webrtc program can receive it without changing the logic.
Maybe rtspsrc, v4l2src ...

File: 3rdparty/README.md [hunk 3]
```diff
@@ -15,6 +17,27 @@ This documents provides guidelines to install board SDK required by this library
 
 FILE capturer is a dummy capturer that read sample frames from [resources/frames/]. It doesn't require user to manually install any 3rdparty SDK.
 
+## QCS610
+
+QCS610 receive and retrieve audio and video from the device through the Gstreamer pipeline. It doesn't require user to manually install any 3rdparty SDK.
+Please configure the Qualcomm build environment first.
+Replacing CMake configuration file.
+```
+export OE_CMAKE_TOOLCHAIN_FILE=3rdparty/QCS610/OEToolchainConfig.cmake
```
[the remark this answers is not in the document]
Comment (chenln1124) [reply 3rdparty/README.md:26]: > If it's a yocto based build system, should it be better if user have a recipe for this project?

It is not compiled on Yocto, only cross-compiled on Ubuntu. So there is no Yocto recipe file written for it.

**Pull request merged**